airflow/cli/commands/connection_command.py

```diff
@@ -238,39 +238,24 @@ def connections_delete(args):
 
 @cli_utils.action_logging
 def connections_import(args):
-    """Imports connections from a given file"""
+    """Imports connections from a file"""
     if os.path.exists(args.file):
         _import_helper(args.file)
     else:
         raise SystemExit("Missing connections file.")
 
 
 def _import_helper(file_path):
-    """Helps import connections from a file"""
-    connections_dict = load_connections_dict(file_path)
+    """Load connections from a file and save them to the DB. On collision, skip."""
+    connections_dict = _parse_secret_file(file_path)
```

Comment:
Wait, this shouldn't change, it should still call `load_connections_dict`, but we should change _that_ function to call.


_wait_. Looking at load_connections_dict, I see it is _already_ returning a dict with values of Connection objects!

Did this cli command actually work?!

Reply:
Nope, I introduced a bug at the last minute. The values of the dict were indeed Connection objects, but I was treating them as dictionaries (and my test didn't catch it). I can change this back to `load_connections_dict`, but I thought this was more straightforward.